canonical-websites/usn.ubuntu.com
# Add ./run for local development

## Done
Added the run script. Set up the sass and set the development port to 8020.

## QA
- Run `./run clean` to reset the build files
- Run `./run` _may take some minutes_
- Check the site starts correctly
- In a new terminal run `./run watch` and edit the brand-color in `_settings_color.scss` and save.
- Wait for the site to rebuild and then check the color updated

## Details
Fixes https://github.com/canonical-websites/usn.ubuntu.com/issues/3
